groups/rds/main.tf

```diff
@@ -1,10 +1,10 @@
 terraform {
-  required_version = ">= 0.13.0, < 0.14"
+  required_version = "~> 1.3.0"
```

Comment:
We should look to use lower and upper bound for version constraints.

E.g.:
```
  required_version = ">= 1.3.0, < 2.0.0"
```

Reply:
Completed as required